File: example/src/main/java/com/amqtech/opensource/appintroexample/ui/fragment/PermissionTabsFragment.java
package com.amqtech.opensource.appintroexample.ui.fragment;

import android.graphics.Color;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v4.view.ViewPager;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.amqtech.opensource.appintroexample.ui.permsTabs.PermsPagerAdapter;
import com.amqtech.opensource.appintroexample.util.TabLayout;
import com.github.paolorotolo.appintroexample.R;

/**
 * Created by andrew on 11/17/16.
 */

public class PermissionTabsFragment extends Fragment {

    ViewPager pager;
    PermsPagerAdapter adapter;
    TabLayout tabs;
    CharSequence Titles[] = {"Layout 1", "Layout 2"};
    int Numboftabs = 2;

    public PermissionTabsFragment() {
        //required empty constructor
    }

    @SuppressWarnings({"ConstantConditions", "deprecation"})
    @Override
    public void onActivityCreated(Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        adapter = new PermsPagerAdapter(getChildFragmentManager(), Titles, Numboftabs);

        pager = getView().findViewById(R.id.permPager);
        pager.setAdapter(adapter);

        tabs = getView().findViewById(R.id.permTabs);
        tabs.setBackgroundColor(Color.parseColor("#1976D2"));
        tabs.setDistributeEvenly(true);
        tabs.setCustomTabColorizer(new TabLayout.TabColorizer() {
            @Override
            public int getIndicatorColor(int position) {
                return Color.parseColor("#1976D2");
            }
        });
        tabs.setViewPager(pager);
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_permission_tabs, container, false);
    }
}

File: example/src/main/java/com/github/appintro/example/ui/fragment/PermissionTabsFragment.kt
package com.github.appintro.example.ui.fragment

import android.graphics.Color
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.appintro.example.ui.permsTabs.PermsPagerAdapter
import com.github.paolorotolo.appintroexample.R
import kotlinx.android.synthetic.main.fragment_permission_tabs.*

class PermissionTabsFragment : Fragment() {
    var titles = arrayOf("Layout 1", "Layout 2")

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)

        permPager.adapter = PermsPagerAdapter(childFragmentManager, titles)

        with(permTabs) {
            setDistributeEvenly(true)
            setBackgroundColor(Color.parseColor("#1976D2"))
            setCustomTabColorizer { Color.parseColor("#1976D2") }
            setViewPager(permPager)
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? =
            inflater.inflate(R.layout.fragment_permission_tabs, container, false)
}